sys/include/net/gnrc/tcp/tcb.h

```diff
@@ -40,47 +39,52 @@ extern "C" {
 #endif
 
 /**
+ * @brief Size of the tcbs internal message queue
+ */
+#define GNRC_TCP_TCB_MSG_QUEUE_SIZE (8U)
+
+/**
  * @brief transmission control block of gnrc_tcp
  */
 typedef struct _transmission_control_block {
-    uint8_t address_family;                    /**< Address Family of local_addr and peer_addr */
+    uint8_t address_family;   /**< Address Family of local_addr and peer_addr */
 #ifdef MODULE_GNRC_IPV6
```

Comment:
I want TCP compileable without a network layer, thats the reason why everything address related is surrounded with the #ifdef.